spring-cloud-task-docs/src/main/asciidoc/appendix-cloud-foundry.adoc

```diff
@@ -0,0 +1,34 @@
+
+[[appendix-cloud-foundry]]
+== Running a Task App on Cloud Foundry
+
+This section will discuss how to deploy a Spring Cloud Task application onto
+the Cloud Foundry platform.
+
+There are 3 steps to deploying your Spring Cloud Task application:
+
+. We must push your application to the platform using the `cf push command`.
+This command pushes your jar to the Cloud Foundry platform.
+
+. We need to stop the application from running once the push is complete.
```

Comment:
why is the `stop` command not mentioned here https://docs.cloudfoundry.org/devguide/using-tasks.html ?